addOns/custompayloads/src/main/java/org/zaproxy/zap/extension/custompayloads/CustomPayloadsOptionsPanel.java

```diff
@@ -46,15 +49,33 @@ public CustomPayloadsOptionsPanel() {
         gbc.anchor = GridBagConstraints.LINE_START;
         gbc.fill = GridBagConstraints.BOTH;
 
-        this.add(new JLabel(OPTIONS_TITLE), gbc);
+        this.add(getTitleLabel(), gbc);
         gbc.weighty = 1.0;
         this.add(tablePanel, gbc);
     }
 
+    private static JLabel getTitleLabel() {
+        if (titleLabel == null) {
+            titleLabel = new JLabel(OPTIONS_TITLE);
```

Comment:
Should be initialized in the constructor, this will be called always and the variable will never be null thereafter.